code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm

```diff
@@ -0,0 +1,613 @@
+
+var/global/list/ts_ckey_blacklist = list()
+var/global/ts_count_dead = 0
+var/global/ts_count_alive_awaymission = 0
+var/global/ts_count_alive_station = 0
+var/global/ts_death_last = 0
+var/global/ts_death_window = 9000 // 15 minutes
+var/global/list/ts_spiderlist = list()
+var/global/list/ts_egg_list = list()
+var/global/list/ts_spiderling_list = list()
+
+// --------------------------------------------------------------------------------
+// --------------------- TERROR SPIDERS: DEFAULTS ---------------------------------
+// --------------------------------------------------------------------------------
+// Because: http://tvtropes.org/pmwiki/pmwiki.php/Main/SpidersAreScary
+
+/mob/living/simple_animal/hostile/poison/terror_spider/
+	// Name / Description
+	name = "terror spider"
+	var/altnames = list()
+	var/name_usealtnames = 0 // if 1, spiders use their randomized names, if not they're all "<color> terror"
+	desc = "The generic parent of all other terror spider types. If you see this in-game, it is a bug."
+	var/egg_name = "terror eggs"
+
+	// Icons
+	icon = 'icons/mob/terrorspider.dmi'
+	icon_state = "terror_red"
+	icon_living = "terror_red"
+	icon_dead = "terror_red_dead"
+
+	// Health
+	maxHealth = 120
+	health = 120
+
+	// Melee attacks
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	attacktext = "bites"
+	attack_sound = 'sound/weapons/bite.ogg'
+	poison_type = "" // we do not use that silly system.
+
+	// Movement
+	//anchored = 1 // stops people dragging them around -- requires testing *****
```

Comment:
it might require testing
but it certainly is commented out now and NO ADDING COMMENTED OUT CODE